modules/ROOT/pages/associate-environments.adoc

```diff
@@ -15,10 +15,16 @@ To associate an environment to a Runtime Fabric:
 
 == Production Environments
 
-When selecting a production environment, a dialog appears to verify Runtime Fabric has been configured to operate in a production setting. This includes proper hardware configuration, log forwarding, alerting, and familiarity with possible failure modes.
+When selecting a production environment, a dialog is displayed that verifies Runtime Fabric has been configured to operate in a production setting. This includes proper hardware configuration, log forwarding, alerting, and possible failure modes.
```

Comment:
"a dialog is displayed that verifies" this word order sounds odd to me, shouldn't it be: 
"a dialog that verifies Runtime Fabric has been configured to operate in a production setting is displayed."

Another option could be:
"Runtime Manager displays a dialog that verifies Runtime Fabric is configured to operate in a production setting"